flocker/acceptance/testtools.py

```diff
@@ -222,7 +225,12 @@ def _stop_acceptance_cluster():
     agent_nodes = filter(None, agent_nodes_env_var.split(':'))
 
     if control_node and agent_nodes:
-        stop_cluster(control_node, agent_nodes)
+        return succeed(sync_perform(
+            dispatcher,
+            stop_cluster(control_node, agent_nodes)
+        ))
+    else:
+        return succeed(None)
```

Comment:
Document the return values in the docstring. Also, I don't understand why this returns a Deferred now.